Dockerfile

```diff
@@ -28,4 +28,8 @@ RUN ant clean all && \
     rm -rf lib && \
     rm build.xml
 
-ENTRYPOINT ["./docker_helper.sh"]
\ No newline at end of file
+RUN mkdir /usr/working
```

Comment:
Should this mkdir be moved to a previous run command in order to avoid an additional layer, and for more efficient caching?